scripts/install_dependencies.bash

```diff
@@ -4,76 +4,29 @@ set -e
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 REPO_DIR=$(dirname "$SCRIPT_DIR")
 
-install_python_dependencies()
-{
-    echo ""
-    echo "Installing Python3.8 and other packages"
-    echo ""
-    # Ensure we have Python3.8 and pip3
-    sudo apt-get install -qq python3.8 python3.8-dev python3-tk python3-pip python3-pil python3-pil.imagetk
-
-    echo ""
-    echo "Installing the required dependencies for catkin for Python3.8"
-    echo ""
-    python3.8 -m pip -q install catkin_pkg rospkg
-
-    echo ""
-    echo "Installing the required dependencies for GeoScenario Server for Python3.8"
-    echo ""
-    python3.8 -m pip install numpy scipy glog matplotlib py_trees==0.7.6 antlr4-python3-runtime==4.9.3 tk sysv-ipc antlr-denter
-}
-
-install_lanelet2_python38()
-{
-    echo ""
-    echo "Installing Lanelet2 library from source for Python3.8..."
-    echo ""
-    # Ensure catkin is installed
-    sudo apt-get -qq install python-catkin-tools
-    cd $REPO_DIR
-    # In the case the repository was not cloned recursively
-    git submodule update --init
-
-    # temporarily switch to Python3.8 for the catkin build
-    alias python=/usr/bin/python3.8
-    alias python3=/usr/bin/python3.8
-
-    mkdir -p $REPO_DIR/catkin_ws/src
-    ln -sfn $REPO_DIR/Lanelet2 $REPO_DIR/catkin_ws/src/Lanelet2
-
-    source "$SCRIPT_DIR/catkin_utils.bash"
-
-    local CATKIN_DIR="$REPO_DIR/catkin_ws"
-
-    create_catkin_workspace $CATKIN_DIR
-
-    build_catkin_workspace $CATKIN_DIR --clean
+echo ""
+echo "Upgrading packages"
+echo ""
+sudo apt-get update && sudo apt-get upgrade -y
```

Comment:
The `apt-get upgrade` here is a bit weird, since it will update packages completely unrelated to geoscenarioserver.